inst/rmarkdown/templates/tufte_book/template.yaml

```diff
@@ -0,0 +1,5 @@
+name: Tufte Book (PDF)
+description: >
+  Template for creating a book according to the style of 
+  Edward R. Tufte and Richard Feynman.
+create_dir: false
```

Comment:
I think this should be TRUE (since we have to copy a bunch of files side by side with the skeleton.Rmd)